miniapps/blocksolvers/block_solvers_comparison.cpp

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+// Description:  This miniapp compares various linear solvers for the saddle
```

Comment:
It would be nice to list the solvers being compared (one or two words per solver)